barrantes_sofiaanne/index.html

```diff
@@ -0,0 +1,170 @@
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/564x/ac/f5/13/acf513e438c3be05d8694ff1bb6f4712.jpg" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/736x/52/b4/c5/52b4c5f65a92e2167a89a0621bdc2a8e.jpg" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://static.independent.co.uk/s3fs-public/thumbnails/image/2015/06/10/10/jCubeDog.jpg?width=1200" />
+        </div>
+        <div class="gallery-item">
+            <img src="https://i.pinimg.com/564x/5d/29/93/5d2993b3ad0f1d37c630d3132114a1c0.jpg" />
+        </div>
+    </div>
+    <div class="contact">
+        <h2>Contact Me</h2>
+        <form>
+            <div class="email">
+                <label for="email">Email:</label>
+                <input type="email" id="email" name="email" />
+            </div>
+            <div class="Name">
+                <label for="Name">Full Name:</label>
+                <input type="text" id="Name" name="Name" />
+            </div>
+            <div class="subject">
+                <label for="subject">Subject:</label>
+                <input type="text" id="subject" name="subject">
+            </div>
+            <div class="age">
+                <label for="age">Age:</label>
+                <input type="number" id="age" name="age">
+            </div>
+            <div class="copy">
+                <label for="copy">Send me a copy</label>
+                <input type="checkbox" name="copy" id="copy">
+            </div>
+            <div class="message">
+                <label for="message">Message:</label>
+                <textarea id="message" name="message" rows="5" cols="33">
+                </textarea>
+            </div>
+            <div class="submit">
+                <button type="submit">Submit</button>
+            </div>
+        </form>
+    </div>
+    <div class="goals">
+        <div class="split-right">
+            <div class="section-container">
+                <div class="slide">
+                    <h2>My Goals</h2>
+                    <ul>
+                        <li>To graduate</li>
+                        <li>Receive a job offer that is good for a new grad</li>
+                        <li>Pass the landerized exam</li>
+                        <li>Good health and stable financially</li>
+                    </ul>
+                    <h2>Comment</h2>
+                    <ul>
+                        <li> May your come true! -Precious </li>
+                        <li> Great Job! - Eliza</li>
+                        <li>Goodluck on your goals. :D - Mikaella</li>
+                        <li>Praying for you! - Alona</li>
+                    </ul>
+                </div>
+                <div class="comm">
+                    <h2>Comment Form</h2>
+                    <form id="commentForm">
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5